andela/travel_tool_front
# #160724471 Confirmation dialog for approval

#### What does this PR do?
Shows confirmation modal to manager before Approval or Rejection action

#### Description of Task to be completed?
- add confirmation modal to requests modal
- show different confirmation dialog when approve or reject button is clicked
- move action button to confirmation dialog

#### How should this be manually tested?
- Clone the repo - https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Install dependencies and start the app - make start
- Create a new request on My Requests tab and ensure you're listed as your manager
- Log in, click Requests tab on the left sidebar and select My Approvals
- Click an open request or create a new
- Any button out of Approve and Reject can be clicked to update the status of the request
- Click an open request on My Requests && My Approvals page to display modal
- Click on an approved or rejected request and click both buttons

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160724471](https://www.pivotaltracker.com/story/show/160724471)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-27 at 4 36 13 pm" src="https://user-images.githubusercontent.com/23608060/46157395-83057280-c273-11e8-9442-9fa7cddd6abe.png">

#### Questions:
